database/src/main/resources/db/migration/R__GTFS_query.sql

```diff
@@ -392,9 +391,9 @@ BEGIN
 
   --RAISE NOTICE 'yli jäi d1: % ja d2: % vuoroa', array_length(d1_trip_ids, 1), array_length(d2_trip_ids, 2);
 
-  route_changes."route-short-name" := "route-short-name";
-  route_changes."route-long-name" := "route-long-name";
-  route_changes."trip-headsign" := "trip-headsign";
+  route_changes."route-short-name" := null; --"route-short-name";
```

Comment:
Are the old values left in comments on purpose?

Reply:
route_changes uses composition type which requires something to be added on those columns.
When we migrate to composite type to own table, this won't be needed anymore. So yes and no :) we don't need those comments but this is going to change soon anyway.